workflow/envs/fuseq_wes.yaml

```diff
@@ -0,0 +1,4 @@
+channels:
+  - conda-forge
+  - bioconda
+  - defaults
```

Comment:
We need to add  "OBSERVER" (or similar) to the Readme that fuseq_wes need to be available on the server, if the user plan to use conda since conda cannot install fuseq-wes.